qp_shogun/shogun/__init__.py

```diff
@@ -28,7 +28,28 @@
     # threads
     'Number of threads': ['integer', '1'],
     }
-outputs = {'Functional Predictions': 'BIOM', 'Taxonomic Predictions': 'BIOM'}
+outputs = {
+    'Functional Predictions - genus, KEGG Modules Coverage': 'BIOM',
+    'Functional Predictions - genus, KEGG Modules': 'BIOM',
+    'Functional Predictions - genus, KEGG Pathways Coverage': 'BIOM',
+    'Functional Predictions - genus, KEGG Pathways': 'BIOM',
+    'Functional Predictions - genus, KEGG': 'BIOM',
+    'Functional Predictions - genus, Normalized': 'BIOM',
```

Comment:
I think for now we can omit the 'genus' and 'strain' levels of the `Functional Predictions` BIOM types, as we're restricting this manually to 'species' level  on line 223 of shogun.py